src/Development/IDE/Spans/Type.hs

```diff
@@ -34,9 +35,11 @@ data SpanInfo =
             -- any. This can be useful for accessing a variety of
             -- information about the identifier such as module,
             -- locality, definition location, etc.
+           ,spaninfoDocs :: ![T.Text]
+           -- ^ Documentation for the element
            }
 instance Show SpanInfo where
-  show (SpanInfo sl sc el ec t n) =
+  show (SpanInfo sl sc el ec t n _docs) =
```

Comment:
Given that the `Show` instance is intended for debugging, we should include all fields in it.